packages/repository-tests/src/crud-test-suite.ts

```diff
@@ -50,15 +50,40 @@ export function crudRepositoryTestSuite(
       }),
     );
 
+    let testFiles = [];
+
     const testRoot = path.resolve(__dirname, 'crud');
-    let testFiles = fs.readdirSync(testRoot);
+    testFiles = fs.readdirSync(testRoot);
     testFiles = testFiles.filter(function(it) {
       return (
         !!require.extensions[path.extname(it).toLowerCase()] &&
         /\.suite\.[^.]+$/.test(it)
       );
     });
 
+    // relations folder tests
+    const folders = ['acceptance', 'integration'];
```

Comment:
All tests in `repository-tests` are meant to be acceptance-level. Let's keep things simple(r) and "promote" any tests that are considered to be integration-level in their original place to become acceptance-level here.